### ISSUE TITLE
Kill job while running?

### ISSUE THREAD

**Issue by drgroot** [75lb/handbrake-js#19]
say I'm in the progress part, as such

```
.on("progress", function(progress){
       if(progress.eta == too_long_for_me ){
         // kill job
       }
     })
``` 
Is it possible to kill the process?

**Comment by 75lb** [comment]
actually, that's a good question and i'm surprised noone mentioned it before.. no you can't.

We need a `.kill()` method on the [Handbrake](https://github.com/75lb/handbrake-js#hbjshandbrake--eventemitter) object which calls [.kill()](https://nodejs.org/dist/latest-v6.x/docs/api/child_process.html#child_process_child_kill_signal) on the underlying child process.. do you have time to send a patch?  If not, i'll add it to my todo list.

**Comment by drgroot** [comment]
hmm sure, i'll try and look at it, new to javascript, but I'm down to help!

**Comment by 75lb** [comment]
fixed and released in v2.1.0 👍
